docs/topics/BrowserPlugin.adoc

```diff
@@ -66,9 +66,10 @@ The KeePassXC-Browser extension lets you automatically populate the entries from
 
 2. Open your web browser. The KeePassXC-Browser extension icon in your browser window will change based on its connection state. The figure below shows the different states.
 +
-*(A)* KeePassXC is not running or is disconnected +
-*(B)* Connected to KeePassXC, but database is locked +
-*\(C)* Connected to KeePassXC and ready to use
+*(A)* KeePassXC is not running or is disconnected. +
+*(B)* KeePassXC is running, but KeePassXC Browser Extension is not connected to the current database. +

```

Comment:
s/KeePassXC Browser Extension/the KeePassXC-Browser extension/
I do not understand which database is "the current" one.

Reply:
Uh, I think you can take a guess 😉